resources/views/security/signup.twig

```diff
@@ -10,16 +10,27 @@
 
         <form method="post" action="{{ url('user_create') }}">
             <label class="text-white text-sm">Email</label>
-            <input type="email" name="email" placeholder="you@example.com">
+            <input type="email" name="email" placeholder="you@example.com" value="{% if email %}{{ email | escape }}{% endif %}">
             <label class="text-white text-sm">Password</label>
             <input type="password" name="password" placeholder="•••••••">
-            <div class="text-sm my-4 flex items-center">
-                <input id="coc" class="mr-2" name="coc" type="checkbox">
-                <label for="coc" class="text-white">I agree to abide by the <a href="{{ site.coc_link }}" target="_blank" class="text-brand text-bold text-underline" rel="noopener noreferrer">Code of Conduct</a></label>
-                <input id="privacy" class="mr-2" name="privacy" type="checkbox">
-                <label for="privacy" class="text-white">I have read and understand the <a href="{{ url('privacy') }}" target="_blank" class="text-brand text-bold text-underline" rel="noopener noreferrer">Privacy Policy</a></label>
-            </div>
+            <ul class="list-reset">
+                <li>
+                    <label for="coc" class="text-white">
+                        <input id="coc" class="mr-2" name="coc" type="checkbox"> I agree to abide by the <a href="{{ site.coc_link }}" target="_blank" class="text-brand text-bold text-underline" rel="noopener noreferrer">Code of Conduct</a>
```

Comment:
Wrapping the checkboxes within the label improves accessibility - now you can click on the entire label to check/uncheck the checkbox.